teachertool/src/components/CriteriaInstanceDisplay.tsx

```diff
@@ -1,11 +1,10 @@
+import css from "./styling/CriteriaInstanceDisplay.module.scss";
 import { getCatalogCriteriaWithId } from "../state/helpers";
 import { CriteriaInstance, CriteriaParameterValue } from "../types/criteria";
 import { logDebug } from "../services/loggingService";
 import { setParameterValue } from "../transforms/setParameterValue";
 import { classList } from "react-common/components/util";
 import { getReadableBlockString, splitCriteriaTemplate } from "../utils";
-// eslint-disable-next-line import/no-internal-modules
```

Comment:
Why does moving this import to the top of the file allow for getting rid of this?

Reply:
We disabled this eslint warning in config some time ago, so this line wasn't needed anymore.